### PR TITLE
TW-1698: group details in small in width

### PR DESCRIPTION
**Pull request by sherlockvn**
#1698 

### Solution:
- add ratio for messageBubble width

###

https://github.com/linagora/twake-on-matrix/assets/43041967/dffa4741-6a29-4208-91cd-5c0d8dfd9e05

### CODE REVIEW

**Comment by Te-Z** [comment]
Any ticket for this subjet ?

**Comment by hieutbui** [comment]
Cause it change the `maxWidth` of `messageBubble`, I worry about the case when change system fontsize as we fixed at #1669. Can you test this case?

**Comment by sherlockvn** [comment]
> Cause it changes the `maxWidth` of `messageBubble`, I worry about the case when change system fontsize as we fixed at #1669. Can you test this case?


https://github.com/linagora/twake-on-matrix/assets/43041967/54cdff0a-eeb3-4bce-8cd8-26a700894c60

**Comment by sherlockvn** [comment]
> Cause it changes the `maxWidth` of `messageBubble`, I worry about the case when change system fontsize as we fixed at #1669. Can you test this case?

For the case of big text:

https://github.com/linagora/twake-on-matrix/assets/43041967/10f992d2-dd94-4244-af32-bac1fc7dbc5a

**Review by hieutbui — APPROVED** [review]

**Review by Te-Z — APPROVED** [review]

File: lib/pages/chat_adaptive_scaffold/chat_adaptive_scaffold_builder.dart [hunk 1]
```diff
@@ -70,8 +69,8 @@ class ChatAdaptiveScaffoldBuilderController
 
   @override
   Widget build(BuildContext context) {
-    const breakpoint = ResponsiveUtils.minTabletWidth +
-        MessageStyle.messageBubbleDesktopMaxWidth;
+    final responsiveUtils = ResponsiveUtils();
+    final breakpoint = responsiveUtils.getMinDesktopWidth(context);
```
Comment (nqhhdev) [inline lib/pages/chat_adaptive_scaffold/chat_adaptive_scaffold_builder.dart:73]: IMO: Create getter func outside builder func

**Pull request merged**